rskj-core/src/main/java/co/rsk/peg/union/constants/UnionBridgeMainNetConstants.java

```diff
@@ -19,43 +20,43 @@ private UnionBridgeMainNetConstants() {
         btcParams = NetworkParameters.fromID(NetworkParameters.ID_MAINNET);
 
         // TODO: Replace with actual address
-        // seed: UNION_BRIDGE_ADDRESS
-        unionBridgeAddress = new RskAddress("5988645d30cd01e4b3bc2c02cb3909dec991ae31");
+        unionBridgeAddress = new RskAddress("0000000000000000000000000000000000000000");
 
         // TODO: Replace with actual initial locking cap value and increments multiplier
         BigInteger oneRbtc = BigInteger.TEN.pow(18); // 1 RBTC = 1000000000000000000 wei
         initialLockingCap = new Coin(oneRbtc).multiply(BigInteger.valueOf(300)); // 300 rbtc
         lockingCapIncrementsMultiplier = 2;
 
         // TODO: Replace with actual authorizers
-        List<ECKey> changeUnionBridgeContractAddressAuthorizers = Collections.singletonList(
-            // seed: unionBridgeAuthorizer
-            ECKey.fromPublicOnly(Hex.decode("041fb6d4b421bb14d95b6fb79823d45b777f0e8fd07fe18d0940c0c113d9667911e354d4e8c8073f198d7ae5867d86e3068caff4f6bd7bffccc6757a3d7ee8024a"))
-        );
+        List<ECKey> changeUnionBridgeContractAddressAuthorizers = Stream.of(
```

Comment:
Should we be setting this value for mainnet?

Reply:
Makes no difference. I'll remove it then.

Reply:
Removed